test/ci-test-weather.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
it's not so simple, because you can define a reference based on the uri of
the schema, but then:
- it happens that the uri is not referenciable, which means that you cannot
retrieve it (see the http://json-schema.org/geojson/geometry.json)
- it happens that schemas that you referenced are not 100% valid because
they don't include id definition that help in the resolution of included
schema (see the the issues i had with validation of nezta model)

i.e. it only works in a perfect world, which we don't have :S